template/themesrc/includes/api/index.php

```diff
@@ -1,2 +0,0 @@
-<?php
```

Comment:
Met die index.php reden zat deze er ook in. Dit doen sommige plugins ook, overal een lege index.php in plaatsen om misconfiguraties op te vangen. zo kun je nooit een file list te zien krijgen. 
Dit zal in ons geval niet zo relevant zijn, maar dat is waar het vandaan komt